row col-4 .dillan html/css add

teaglebuilt opened the pull request:
For some reason my container size was not the same height as yours. Im not quite sure why. could not find where the height and width setting was in css mixed with the css framework/col-4.

accidently added a couple files that i had to ignore. .firebasearc and ds store. 

If i did anything else wrong plz let me know.

teaglebuilt committed:
savvy

teaglebuilt committed:
Delete .DS_Store

davi3blu3 committed:
Update .firebaserc

davi3blu3 reviewed (approved):
Looks good! Approving everything except the changes to .firebaserc which I should have gitignored originally - that's unique to each of us.

I haven't seen the linear-gradient property used with additional styles after the parenthesis are closed? Not sure what that's doing exactly, but I'm curious to hear where you were going with that. Other than that your syntax looks good - nice job!

teaglebuilt commented:
hey so the additional styles after the linear gradient quote are actually
for the linear gradient. The size of the linear-gradients pattern. If i
didnt give it a size it would just appear black.

I also updated my css and fixed the height so its equal to your container
and filled the portfolio and facebook link as well. Although when i sent it
to github it never showed up. i tried it again and the terminal said i had
basically already done it . See you at class

On Sun, Nov 5, 2017 at 11:26 PM, Dave Harned <notifications@github.com>
wrote:

> Merged #1 <https://github.com/davi3blu3/student-showcase/pull/1>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/davi3blu3/student-showcase/pull/1#event-1326918599>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARyAFyT3vfvlNQXSOnkbCsBgKxZm1qQNks5szpiQgaJpZM4QSaJT>
> .
>
